soc/snps/nsim/arc_classic/em/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# SPDX-License-Identifier: Apache-2.0
+
+if(COMPILER STREQUAL gcc)
+  # GNU compiler options
+  zephyr_compile_options(-mcpu=${GCC_M_CPU})
+
+  zephyr_compile_options(-mno-sdata)
+  zephyr_compile_options_ifdef(CONFIG_CPU_ARCEM -mmpy-option=wlh1)
+  zephyr_compile_options_ifdef(CONFIG_FPU -mfpu=fpuda_all)
+
+else()
+  # MWDT compiler options
+  zephyr_compile_options_ifdef(CONFIG_SOC_NSIM_EM -arcv2em -core3 -Xdiv_rem=radix2
+			       -Xmpy_option=mpyd -Xbitscan -Xswap -Xbarrel_shifter
+			       -Xshift_assist -Xdsp2 -Xdsp_complex
+			       -Xdsp_divsqrt=radix2 -Xdsp_itu -Xdsp_accshift=full
+			       -Xfpus_div -Xfpu_mac -Xfpuda -Xfpus_mpy_slow
+			       -Xfpus_div_slow -Xbitstream -Xtimer0 -Xtimer1)
+
+  #FIXME: CONFIG_SOC_NSIM_EM7D_v22 ?
```

Comment:
Why 'fixme' I don't recall metaware was supported for this variant in the old structure. If you think it should be added, please create an issue for it instead of this 'fixme'.

Reply:
Done